### ISSUE TITLE
lerna import --flatten corrupts PDF files

### ISSUE THREAD

**Issue by a-sync** [lerna/lerna#2190]
## Expected Behavior
PDF files imported into a monorepo should not be modified. (doesn't happen if --flatten argument is not defined)

## Current Behavior
After the import the PDF file is transformed into a text file containing the contents of the original PDF.

## Possible Solution
PDF's should be handled as binary when importing and applying commits. (?)

## Steps to Reproduce (for bugs)
1. import a repo which has a PDF file in it with the --flatten option
2. check the PDF within the monorepo

A simple, all steps included repro: https://github.com/a-sync/lerna-import-pdf-test.git

## Context
Had a few PDF files in one of the packages we imported and got baffled by how they ended up as plain text files.

## Your Environment

| Executable | Version |
| ---: | :--- |
| `lerna --version` | 3.15.0 |
| `npm --version`  | 6.9.2 |
| `node--version` | 10.15.1 |

| OS | Version |
| --- | --- |
| Windows 10 | 1809 |

**Comment by tomyam1** [comment]
This also affects docx files.

Might be related - text files which are not UTF8 encoded are treated as UTF8 and thus corrupted.
(A use case for having them in the repo is test fixtures)

**Comment by a-sync** [comment]
tldr; repro is above in the original issue post if anyone actually cares to fix this.
